locales/it/json.json

```diff
@@ -609,28 +609,28 @@
     "Thanks for your continued support. We've attached a copy of your invoice for your records. Please let us know if you have any questions or concerns.": "Grazie per supportarci ancora. Ti abbiamo allegato una copia della fattura per il tuo pagamento. Scrivici se hai domande o dubbi, grazie.",
     "Thanks,": "Grazie,",
-    "The :attribute must be a valid role.": ":Attribute deve avere un ruolo valido.",
-    "The :attribute must be at least :length characters and contain at least one number.": ":Attribute deve contenere almeno :length caratteri ed un numero.",
-    "The :attribute must be at least :length characters and contain at least one special character and one number.": ":Attribute deve contenere almeno :length caratteri, un carattere speciale ed un numero.",
-    "The :attribute must be at least :length characters and contain at least one special character.": ":Attribute deve contenere almeno :length caratteri ed un carattere speciale.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character and one number.": ":Attribute deve contenere almeno :length caratteri, una maiuscola e un numero.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character and one special character.": ":Attribute deve contenere almeno :length caratteri, una maiuscola e un carattere speciale.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character, one number, and one special character.": ":Attribute deve contenere almeno :length caratteri, una maiuscola, un numero e un carattere speciale.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character.": ":Attribute deve contenere almeno :length caratteri e una maiuscola.",
-    "The :attribute must be at least :length characters.": ":Attribute deve contenere almeno :length caratteri.",
-    "The :attribute must contain at least one letter.": ":Attribute deve contenere almeno una lettera.",
-    "The :attribute must contain at least one number.": ":Attribute deve contenere almeno un numero.",
-    "The :attribute must contain at least one symbol.": ":Attribute deve contenere almeno un carattere speciale.",
-    "The :attribute must contain at least one uppercase and one lowercase letter.": ":Attribute deve contenere almeno un carattere maiuscolo ed uno minuscolo.",
-    "The :resource was created!": ":Resource creato/a!",
-    "The :resource was deleted!": ":Resource eliminato/a!",
-    "The :resource was restored!": ":Resource ripristinato/a!",
-    "The :resource was updated!": ":Resource aggiornato/a!",
+    "The :attribute must be a valid role.": ":attribute deve avere un ruolo valido.",
```

Comment:
There is no need to change the first letter - when accepting the PR, the script will automatically replace it.

Why this happens is described [here](https://github.com/Laravel-Lang/lang/discussions/2342).